optuna/study.py

```diff
@@ -306,12 +306,27 @@ def trials_dataframe(self, include_internal_fields=False):
 
             Get an objective value and a value of parameter ``x`` in the first row.
 
-            >>> df = study.trials_dataframe()
-            >>> df
-            >>> df.value[0]
-            0.0
-            >>> df.params.x[0]
-            1.0
+            .. testsetup:: *
+
+                import optuna
+
+                def objective(trial):
+                    trial.report(trial.number)
+                    trial.suggest_uniform('x', float(trial.number), float(trial.number))
+                    return trial.number
```

Comment:
We may want to change the scenario. Returning a trial number is somewhat unrealistic as a user scenario.

Reply:
Addressed this comment as well in https://github.com/optuna/optuna/pull/702/files#r350058631.